docs/data/data-grid/overlays/overlays.md

```diff
@@ -34,7 +78,9 @@ The no results overlay is displayed when the data grid has no results after filt
 
 ### Custom component
 
-If you want to customize the no results overlay, a component can be passed to the `noResults` slot and rendered in place. In the following demo, an illustration is added on top of the default "No results found" message.
+If you want to customize the no results overlay, a component can be passed to the `noResults` slot and rendered in place.
+
+In the following demo, an illustration is added on top of the default "No results found" message.
 
 {{"demo": "NoResultsOverlayCustom.js", "bg": "inline"}}
```

Comment:
I can't see the illustration in the dark mode

![image](https://github.com/mui/mui-x/assets/12609561/f902ee09-a201-4bfb-8b03-b4e057eff45c)

Is it only me?

Reply:
@MBilalShafi Good spot, created a separate PR to fix https://github.com/mui/mui-x/pull/13715